config/fixtures.js

```diff
@@ -0,0 +1,93 @@
+import { consensus } from "./contracts/consensus.js";
+import { usernames } from "./contracts/usernames.js";
+import { multiPayment } from "./contracts/multipayment.js";
+
+const fixtureConfig = {
+    "transfer": {
+        recipientAddress: "0x6F0182a0cc707b055322CcF6d4CB6a5Aff1aEb22",
+        value: "100000000",
+    },
+    "transfer-0": {
+        nonce: "123",
+        recipientAddress: "0xb693449adda7efc015d87944eae8b7c37eb1690a",
+    },
+    "transfer-large-amount": {
+        nonce: "17",
+        recipientAddress: "0x6F0182a0cc707b055322CcF6d4CB6a5Aff1aEb22",
+        value: "10000000000000000000",
+    },
+    "vote": {
+        contract: {
+            data: consensus,
+            functionName: "vote",
+            args: ["0xc3bbe9b1cee1ff85ad72b87414b0e9b7f2366763"],
+        },
+        gasLimit: 1000000,
```

Comment:
better to change it to `200000` so the fixtures would represent a transaction that would not run out of gas when actually used :trollface:

Reply:
applies to all the configs